MOSH_README.md

```diff
@@ -0,0 +1,64 @@
+
+    Usage: mosh [options] [user@]host
+    
+          --client=VALUE         Path to client helper on local machine (default:
+                                   "mosh-client")
+          --server=VALUE         Command to run server helper on remote machine
+                                   (default: "mosh-server").
+                                    Example: '--server="mosh-server new -v -c 256"-
+                                   '.
+                                    See https://linux.die.net/man/1/mosh-server for
+                                   more details.
+          --ssh=VALUE            OpenSSH command to remotely execute mosh-server
+                                   on remote machine (default: "ssh").
+                                    Example: ''--ssh="ssh -p 2222"'.
+                                    See https://man.openbsd.org/ssh for more
+                                   details.
+          --predict=VALUE        Controls use of speculative local echo. Defaults
+                                   to 'adaptive' (show predictions on slower links
+                                   and to smooth out network glitches) and can also
+                                   be 'always' or 'never'.
+      -a                         Synonym 'for --predict=always'.
+      -n                         Synonym 'for --predict=never'.
+      -p, --port=VALUE           Use a particular server-side UDP port or port
+                                   range, for example, if this is the only port
+                                   that is forwarded through a firewall to the
+                                   server. Otherwise, mosh will choose a port
+                                   between 60000 and 61000.
+                                    Example: '--port=60000:60100'
+          --help                 Show help.
+          --no-init              Do not send the smcup initialization string and
+                                   rmcup deinitialization string to the client's
+                                   terminal. On many terminals this disables
+                                   alternate screen mode.
+    
+    Exit codes:
+      254 - Invalid command line arguments
+      255 - Initial SSH of Mosh connection setup failed
+      All other values - Exit code returned by remote shell
+
+  
+As you can see, many things are configurable. But for some options you need to know more, for example:
+
+ - To effectively  use `--ssh=` option, you need to know more about OpenSSH command-line syntax ([ssh(1)](https://man.openbsd.org/ssh));
+ - To use `--server=` option you need to know more about `mosh-server` ([mosh-server(1) - Linux man page](https://linux.die.net/man/1/mosh-server))
+ - etc.
```

Comment:
This can be made more concise like this:

For the values taken by `--ssh` see ([ssh(1)](https://man.openbsd.org/ssh)) and for `--server` see [mosh-server(1)](https://linux.die.net/man/1/mosh-server).

Reply:
I've removed this part completely in https://github.com/felixse/FluentTerminal/pull/572 because it's already mentioned in the output of `mosh --help`.